datadog_checks_base/datadog_checks/base/utils/common.py

```diff
@@ -23,6 +24,11 @@ def ensure_unicode(s):
 to_string = ensure_unicode if PY3 else ensure_bytes
 
 
+def round_value(value, sig_digits="0.01", rounding_method=ROUND_HALF_UP):
+    getcontext().rounding = rounding_method
+    return float(Decimal(str(value)).quantize(Decimal(sig_digits)))
```

Comment:
no need to convert to string before passing to decimal

Reply:
You do actually to get proper arithmetic

Reply:
Yeah, there are interesting behaviors when you don't appropriately cast to string. The documentation for Decimal also has all floating point examples in string format - https://docs.python.org/2/library/decimal.html